packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -80,6 +80,11 @@ export class Runtime {
    */
   public static readonly NODEJS_14_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
 
+  /**
+   * The NodeJS 16.x runtime (nodejs16.x)
+   */
+  public static readonly NODEJS_16_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
```

Comment:
Typo
```diff
-  public static readonly NODEJS_16_X = new Runtime('nodejs14.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });
+  public static readonly NODEJS_16_X = new Runtime('nodejs16.x', RuntimeFamily.NODEJS, { supportsInlineCode: true });

```

Reply:
Thank you for catching this! @krk I will follow up with another PR to fix.